abletonosc/clip.py

```diff
@@ -70,27 +70,49 @@ def clip_callback(params: Tuple[Any]) -> Tuple:
             "remove_notes_by_id"
         ]
         properties_r = [
+            ##"end_time" ## doesn't seem to do anything, can't test
             "file_path",
             "gain_display_string",
+            ##"groove", ## if other than None, says "Error handling OSC message: Infered arg_value type is not supported"
+            "has_groove", ## False/True
+            ## is_arrangement_clip - wasn't able to get this to do anything
```

Comment:
That's because arrangement_clips aren't supported yet. Feel free to leave the property in but remove the comment.